skpy/conn.py

```diff
@@ -218,6 +232,10 @@ def __call__(self, method, url, codes=(200, 201, 202, 204, 207), auth=None, head
         debugHeaders = dict(headers)
         if auth == self.Auth.SkypeToken:
             headers["X-SkypeToken"] = self.tokens["skype"]
+            headers["X-Ecs-Etag"] = self.ECS_ETAG
+            headers["X-Skype-Client"] = self.SKYPE_CLIENT
+            headers["X-Skypegraphservicesettings"] = str(self.SKYPE_GRAPH_SETTINGS)
+            headers["User-Agent"] = self.USER_AGENT
```

Comment:
I agree, I support adding the `User-Agent` to globally. 
For the login headers, I recommend to keep the `Mozilla...` headers since they work just fine.
For the directory endpoint, we can reuse the same header values, to maintain consistency.

Reply:
I've added it as a session default for now, as it seems to still work fine through the various flows, and I doubt the existing `python-requests/x.y.z` is any better or worse than this.  I'll revisit if any issues come up.